bundle/edu.gemini.sp.vcs.reg/src/main/scala/edu/gemini/sp/vcs/reg/osgi/Activator.scala

```diff
@@ -1,35 +1,24 @@
 package edu.gemini.sp.vcs.reg.osgi
 
-import edu.gemini.sp.vcs.reg.{VcsRegistrationSubscriber, VcsRegistrar}
+import edu.gemini.sp.vcs.reg.VcsRegistrar
 import edu.gemini.sp.vcs.reg.impl.VcsRegistrarImpl
-import edu.gemini.util.osgi.Tracker._
 
 import org.osgi.framework.{ServiceRegistration, BundleActivator, BundleContext}
-import org.osgi.util.tracker.ServiceTracker
 import java.io.File
 import java.util.logging.Logger
 
 class Activator extends BundleActivator {
   private var reg: ServiceRegistration[VcsRegistrar] = null
```

Comment:
The reference doesn't escape the `Activator` itself and is guaranteed by OSGi to be defined if the `stop` method is called but still you're right, using the `Option` there would still be a bit better.